data/transport missions.txt

```diff
@@ -1951,3 +1951,382 @@ ship "Science Drone"
+			choice
+				`	"Get on my ship!"`
+					goto ship
+				`	"Are you Tod?"`
+			
+			`	"Yeah! Are you here to save me?"`
+			`	"Get on my ship!" you yell, motioning for him to come toward you.`
+			
+			label ship
+			`	The boy runs past you and into your ship. Following closely behind Tod is a large bearded man holding a laser rifle in one hand and a fragmentation grenade in the other.`
+			`	"That's my crew member, you no-good piece of sh-"`
+			`	Before the captain is able to finish his sentence, pipes in the wall next to him explode, blasting hot steam all over the right side of his body, causing him to begin rolling on the floor in horrible screeches of pain. Not wanting to stay connected to the ship any longer than you need to be, you run back into your ship and fly away.`
+
+
+
+mission "Lost Boy 3"
+	landing
+	name `Bring Tod to <planet>`
+	description `Now that Tod's mother knows he is safe, bring him to <destination> so that he may start his mining job to support him and his mother.`
+	destination "Placer"
+	to offer
+		has "Lost Boy 2: done"
+	
+	on offer
+		payment 952
+		conversation
+			`Tod's mother almost tears up at the sight of him as you approach. "Oh, my baby boy is alive!" she exclaims, running up and hugging Tod as his face reddens with embarrassment. "Are you hurt? Are you okay? Do you think you'll be fine?" Tod's mother asks, kissing his cheeks between each question.`
+			`	"I'm fine, Mom," Tod says in an annoyed tone of voice. Some things never change.`
+			`	The mother gives you a handful of credit chips. Upon counting them, you find that they are only worth a total of <payment>. "I know it isn't much for the work you've done, Captain, but it's all I have to give you in return."`
+			choice
+				`	"Don't worry. "`
+				`	"You're lucky I agreed to help you in the first place."`
+					goto money
+			
+			`	"This is just so wonderful. There aren't many people around here who would do such a thing for such little pay."`
+				goto end
+			
+			label money
+			`	"I know, I know, and I'm eternally grateful for that, Captain. There aren't many people around here who would do such a thing for such little pay."`
+			
+			label end
+			`	"Hold on," Tod chimes in. "I still need to get to <planet>."`
+			`	"Of course!" his mother yells out. "Captain, would you please make sure that Tod has a safe passage to <destination>?"`
+			`	"And make sure I don't become a slave this time," Tod quips jokingly.`
+			choice
+				`	"Okay, I'll take Tod to <planet>."`
+					accept
+				`	"Sorry, I'm not headed in that direction."`
+					decline
+	
+	npc
+		government "Pirate"
+		personality staying plunders disables
+		system "Al Dhanab"
+		ship "Falcon (Heavy)" "Cygnet's Slaver"
+	
+	on complete
+		dialog `You wish Tod Copper the best of luck on <planet>. "Thank you, Captain. Hopefully I can repay you one day. As cliche as it sounds, I owe you my life."`
```

Comment:
![image](https://user-images.githubusercontent.com/20871346/32003104-c76d31ac-b963-11e7-82b4-b2982084b8b9.png)
Dropping `Copper` or `on Placer` might make the dialog fit better. Getting a nice wrap and justification can be tricky though, and I've made much worse myself :1st_place_medal: